Makefile

```diff
@@ -143,9 +143,23 @@ gitea-install:
 nexus-install:
 	$(call target_title, "Installing Nexus") \
 	&& make SHARED_SERVICE_KEY=shared-service-sonatype-nexus TF_VAR_nexus_properties_path=../nexus.properties terraform-shared-service-deploy DIR=./templates/shared_services/sonatype-nexus/terraform
-
 # / End migration targets
 
+nexus-cert-install:
+	$(call target_title, "Installing Nexus Cert") \
+	&& make SHARED_SERVICE_KEY=shared-service-nexus-cert terraform-shared-service-deploy DIR=./templates/shared_services/nexus-cert/terraform
+
+nexus-letsencrypt:
+	$(call target_title, "Requesting LetsEncrypt SSL certificate for Nexus") \
+	&& . ./devops/scripts/check_dependencies.sh nodocker,certbot \
+	&& . ./devops/scripts/load_env.sh ./templates/core/.env \
+	&& . ./devops/scripts/load_env.sh ./devops/.env \
+	&& . ./devops/scripts/load_terraform_env.sh ./devops/.env \
+	&& . ./devops/scripts/load_terraform_env.sh ./templates/core/.env \
+	&& pushd ./templates/shared_services/nexus-cert/scripts/ > /dev/null && . ./outputs.sh && popd > /dev/null \
+	&& . ./devops/scripts/load_env.sh ./templates/shared_services/nexus-cert/private.env \
```

Comment:
nit: This may be better as just `.env` as we are leaking a bit of a hack that I had to do for the debugging.